v2/src/components/recipeBoxes/guides.json

```diff
@@ -0,0 +1,79 @@
+{
+  "auth_modes": [
+    {
+      "title": "Email password Login",
+      "icon": "emailPass",
+      "path": "/docs/emailpassword/introduction",
+      "img": "emailPassSS"
+    },
+    {
+      "title": "Social Login",
+      "icon": "social",
+      "path": "/docs/thirdparty/introduction",
+      "img": "socialSS"
```

Comment:
The alignment inside the card is consistent in the designs but different in the output

## Designs
<img width="595" alt="Screenshot 2023-03-13 at 6 50 26 PM" src="https://user-images.githubusercontent.com/18233774/224713949-1c097868-678e-4904-bde5-68d3cb6eaced.png">

## Output
<img width="504" alt="Screenshot 2023-03-13 at 6 50 19 PM" src="https://user-images.githubusercontent.com/18233774/224714034-c8c697c8-18f2-40ca-aadb-7686dcbee735.png">